.claude/skills/deptrac-fixer/examples/03-domain-api-platform.php

```
@@ -0,0 +1,429 @@
+use App\Customer\Application\DTO\CustomerResource;
+use App\Customer\Domain\Repository\CustomerRepositoryInterface;
+use App\Shared\Domain\ValueObject\Ulid;
+
+final readonly class CustomerProvider implements ProviderInterface
+{
+    public function __construct(
+        private CustomerRepositoryInterface $repository
+    ) {
+    }
+
+    public function provide(Operation $operation, array $uriVariables = [], array $context = []): ?CustomerResource
```

PHPMD (2.15.0)

Avoid unused parameters such as '$operation'. (undefined)

(UnusedFormalParameter)